src/main/java/au/com/dmg/fusion/request/adminrequest/AdminRequest.java

```diff
@@ -0,0 +1,56 @@
+package au.com.dmg.fusion.request.adminrequest;
+
+
+import au.com.dmg.fusion.data.ServiceIdentification;
+import au.com.dmg.fusion.request.Request;
+import com.squareup.moshi.Json;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+
+public class AdminRequest implements Request {
+    @Json(name = "ServiceIdentification")
+    private final ServiceIdentification serviceIdentification;
+
+    public ServiceIdentification getServiceIdentification() {
+        return serviceIdentification;
+    }
+
+    public static class Builder{
+        private ServiceIdentification serviceIdentification;
+        public Builder() {
+        }
+        Builder(ServiceIdentification serviceIdentification){
+            this.serviceIdentification = serviceIdentification;
+        }
+
+        public Builder serviceIdentification(ServiceIdentification serviceIdentification){
+            if(serviceIdentification==null){
```

Comment:
If Default is not required (not handled by Satellite), please allow setting to null value so build() can pick up the null error.